component/src/main/resources/org/wso2/carbon/event/input/adapter/mqtt/i18n/Resources.properties

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# WSO2 Inc. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+topic=Topic
+topic.hint=Topic subscribed
+clientId=Client Id
+clientId.hint=client identifier is used by the server to identify a client when it reconnects, It used for durable subscriptions or reliable delivery of messages is required.
+url=Broker Url
+username=Username
+username.hint=Username of the broker (if required)
+password=Password
+password.hint=Password of the broker (if required)
+url.hint=MQTT broker url
+cleanSession=Clean Session
+cleanSession.hint=Persist topic subscriptions and ack positions across client sessions
+keepAlive=Keep Alive (In seconds)
+connection.sslEnabled= SSL Enabled
+connection.sslEnabled.hint=The ssl property of the broker
```

Comment:
change the hint description for connection.sslEnabled.hint as it is not described properly. it's used to  enable the secure connection right?